lib/Parse/ParseStmt.cpp

```diff
@@ -1781,8 +1781,9 @@ Parser::classifyConditionalCompilationExpr(Expr *condition,
           if (!fullCheck) {
             return None;
           }
-          return
-            Context.getModule({ { argumentIdent, UDRE->getLoc() } }) != nullptr;
+
+          auto *M = Context.getModule({ { argumentIdent, UDRE->getLoc() } });
```

Comment:
I agree.  To what extent do we need to load a module to figure out if it is "importable?"  It doesn't seem enough to search for it on disk, and it might depend on further modules that fail to load themselves.